coc/events.py

```diff
@@ -719,12 +719,12 @@ def run_forever(self):
         except KeyboardInterrupt:
             self.close()
 
-    def close(self):
-        """Closes the client and all running tasks."""
-        tasks = {t for t in asyncio.Task.all_tasks(loop=self.loop) if not t.done()}
-        for task in tasks:
-            task.cancel()
-        super().close()
+    # async def close(self):
```

Comment:
We can probably just delete this, no point having commented code, it's always there in the version control if we want to find it anyway